WordPress/WordPressTest/BlogJetpackTests.swift

```diff
@@ -0,0 +1,264 @@
+import XCTest
+import CoreData
+import OHHTTPStubs
+@testable import WordPress
+
+class BlogJetpackTests: CoreDataTestCase {
+
+    // Properties
+
+    private var timeout: TimeInterval = 2.0
+    private var blog: Blog!
+    private var accountService: AccountService!
+    private var blogService: BlogService!
```

Comment:
Can we not simply init these here (lazy vars if needed) instead of doing force-unwraps? A test class is recreated for every test case run so they will always be re-initialized. `setUp` is more for wiping out cached values that might persist and all.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20031/commits/2a6ae7a4841c8f6444723a1c52081eadee63a010.